### ISSUE TITLE
Intersection between test and train categories

### ISSUE THREAD

**Issue by Coder-Liuu** [ju-chen/Efficient-Prompt#15]
Thank you for opening the source code, in the zero sample temporal action recognition, I sent the HMDB51 dataset and actually read `train_split01.txt` and `text_split01.txt,` which I guess are the training set and the test set, and the categories should not be duplicated as I am in the zero sample task. But why are the **action categories duplicated** in these two files? Looking forward to your reply.

**Comment by Coder-Liuu** [comment]
In addition, I split `train_split01.txt` into `train_split02.txt ` and `test_split02.txt` based on the principle of non-duplication of action categories, and I found that the accuracy is very low in the test set. Can you answer this?
